api/handlers_volumes.go

```diff
@@ -47,7 +47,7 @@ func (s *server) VolumeCreateHandler(w http.ResponseWriter, r *http.Request) {
 		req.Encrypted = aws.Bool(false)
 	}
 
-	policy, err := volumeCreatePolicy()
+	policy, err := createPolicy([]string{"ec2:CreateVolume"})
```

Comment:
I have renamed and refactored this function to be generic so that any type of policy can be created by passing the appropriate parameters. If you think this approach is good, we can delete the rest of the policy functions and use the generic function in its place. Please let me know your thoughts.

Reply:
Yeah, that's a good idea - some of these policies are pretty simple so no need to have separate functions. But we might want to keep them separate for more complex ones.
For now let's use this generic function in these Volume handlers and later we can change some of the existing ones.
Maybe we should call it `generatePolicy`